### PR TITLE
Refresh the Splunk documentation page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 1]
```diff
@@ -5,25 +5,43 @@ title: Splunk
...
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Monitoring-Splunk-Api-custom** est utilisé.
 
-* App-Monitoring-Splunk-Api-custom
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
 
-Il apporte les modèles de service suivants :
+| Alias                | Modèle de service                      | Description                                                                                                    |
+|:---------------------|:---------------------------------------|:---------------------------------------------------------------------------------------------------------------|
+| Query-Matches-Number | App-Monitoring-Splunk-Query-Api-custom | Contrôle le nombre de résultats pour une requête donnée. La requête doit commencer par "search ". Par exemple  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:34]: par exemple?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 2]
```diff
@@ -129,75 +153,243 @@ yum install centreon-plugin-Applications-Monitoring-Splunk-Api
...
+| --splunk-retries                           | How many times we should retry queries to splunk. To use in par with the --splunk-wait paramater (Default: 5)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:332]:
```suggestion
| --splunk-retries                           | How many times we should retry queries to Splunk. To use with the --splunk-wait parameter (Default: 5)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 3]
```diff
@@ -129,75 +153,243 @@ yum install centreon-plugin-Applications-Monitoring-Splunk-Api
...
+| --splunk-wait                              | How long (in seconds) should we wait between each retry. To use in par with the --splunk-retries paramater (Default: 2)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:333]:
```suggestion
| --splunk-wait                              | How long (in seconds) we should wait between each retry. To use with the --splunk-retries parameter (Default: 2)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

File: pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 4]
```diff
@@ -5,36 +5,53 @@ title: Splunk
...
+The connector brings the following service templates (sorted by the host template they are attached to):
 
-* App-Monitoring-Splunk-Api-custom
+<Tabs groupId="sync">
+<TabItem value="App-Monitoring-Splunk-Api-custom" label="App-Monitoring-Splunk-Api-custom">
+
+| Service Alias  | Service Template                                | Service Description                                                                                            |
+|:---------------|:------------------------------------------------|:---------------------------------------------------------------------------------------------------------------|
+| Index-Update   | App-Monitoring-Splunk-Index-Update-Api-custom   | Check indexes last update time                                                                                 |
+| Splunkd-Health | App-Monitoring-Splunk-Splunkd-Health-Api-custom | Check the overall health of splunkd. The health of splunkd is based on the health of all features reporting it |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:24]: reporting it?

File: pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 5]
```diff
@@ -5,36 +5,53 @@ title: Splunk
...
-| Index-Update         | App-Monitoring-Splunk-Index-Update-Api   | Check indexes last update time                                                                                  | X       |
-| Query-Matches-Number | App-Monitoring-Splunk-Query-Api          | Check number of results for a query. Query has to start with "search ".Example                                  |         |
-| Splunkd-Health       | App-Monitoring-Splunk-Splunkd-Health-Api | Check the overall health of splunkd. The health of splunkd is based on the health of all features reporting it. | X       |
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias        | Service Template                       | Service Description                                                            |
+|:---------------------|:---------------------------------------|:-------------------------------------------------------------------------------|
+| Query-Matches-Number | App-Monitoring-Splunk-Query-Api-custom | Check number of results for a query. Query has to start with "search ".Example |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:33]: example?

File: pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 6]
```diff
@@ -60,13 +77,13 @@ To use this plugin, you have to configure a user with privileges allowing him to
...
 
-### Monitoring Pack
+### Pack
 
-If the platform uses an *online* license, you can skip the package installation
-instruction below as it is not required to have the pack displayed within the
-**Configuration > Monitoring Connector Manager** menu.
+1. If the platform uses an *online* license, you can skip the package installation
+instruction below as it is not required to have the connector displayed within the
+**Configuration > Monitoring Connectors Manager** menu.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:86]:
```suggestion
**Configuration > Monitoring Connector Manager** menu.
```

File: pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 7]
```diff
@@ -91,11 +108,18 @@ yum install centreon-pack-applications-monitoring-splunk-api
...
+yum install centreon-pack-applications-monitoring-splunk-api
+```
+
 </TabItem>
 </Tabs>
 
-Whatever the license type (*online* or *offline*), install the **Splunk** Pack through
-the **Configuration > Monitoring Connector Manager** menu.
+2. Whatever the license type (*online* or *offline*), install the **Splunk** connector through
+the **Configuration > Monitoring Connectors Manager** menu.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:122]:
```suggestion
the **Configuration > Monitoring Connector Manager** menu.
```

File: pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 8]
```diff
@@ -131,74 +155,241 @@ yum install centreon-plugin-Applications-Monitoring-Splunk-Api
...
+| --splunk-retries                           | How many times we should retry queries to splunk. To use in par with the --splunk-wait paramater (Default: 5)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:332]:
```suggestion
| --splunk-retries                           | How many times we should retry queries to Splunk. To use with the --splunk-wait parameter (Default: 5)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

File: pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md [hunk 9]
```diff
@@ -131,74 +155,241 @@ yum install centreon-plugin-Applications-Monitoring-Splunk-Api
...
+| --splunk-wait                              | How long (in seconds) should we wait between each retry. To use in par with the --splunk-retries paramater (Default: 2)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-splunk-api.md:333]:
```suggestion
| --splunk-wait                              | How long (in seconds) we should wait between each retry. To use with the --splunk-retries parameter (Default: 2)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**